openshift_scalability/content/statefulset-pv-template.json

```diff
@@ -0,0 +1,114 @@
+{
+  "Kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "statefulsetTemplate",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This template will create a statefulset and a headless service.",
+      "tags": ""
+    }
+  },
+  "objects": [
+    {
+      "apiVersion": "v1",
+      "kind": "Service",
+      "metadata": {
+        "name": "nginx${IDENTIFIER}",
+        "labels": {
+          "app": "nginx"
+        }
+      },
+      "spec": {
+        "ports": [
+          {
+            "port": "${PORT}",
+            "name": "web"
+          }
+        ],
+        "clusterIP": "None",
+        "selector": {
+          "app": "nginx${IDENTIFIER}"
+        }
+      }
+    },
+    {
+      "apiVersion": "apps/v1beta1",
+      "kind": "StatefulSet",
+      "metadata": {
+        "name": "web${IDENTIFIER}"
+      },
+      "spec": {
+        "serviceName": "nginx${IDENTIFIER}",
+        "replicas": 2,
```

Comment:
Let's make this a parameter - then we can use set the replicas in the cluster-loader config file if desired.